srtcore/core.h

```diff
@@ -385,7 +388,8 @@ class CUDT
     int64_t maxBandwidth() const { return m_llMaxBW; }
     int MSS() const { return m_iMSS; }
 
-    uint32_t latency_us() const {return m_iTsbPdDelay_ms*1000; }
+    uint32_t peer_latency_us() const {return m_iPeerTsbPdDelay_ms * 1000; }
```

Comment:
Why is `latency_us()` removed?

Reply:
It is not used, and I am concerned about the naming. Likely it should be `tsbPdDelay_ms()`.